lib/ipfs-pinner.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs')
+const path = require('path')
+const { client, PinnerError } = require('@aragonone/ipfs-pinner-client')
+
+const ALREADY_UPLOADED_ERROR = 'File is already uploaded with cid '
+const IPFS_PINNER_ENDPOINT = 'https://ipfs-pinner.backend.aragon.org'
+
+function requiresIpfsUpload(justification) {
+  return !justification.startsWith('ipfs:') && justification.endsWith('.md')
```

Comment:
We don't have to change anything now, but just a heads up: IPFS paths are canonically `/ipfs/<ipld path>` ([us using the `ipfs:` protocol is both somewhat of a random choice, and something the IPFS team was exploring but seems to have given up on](https://github.com/aragon/aragon/pull/1505#discussion_r463547181)).

It's fairly easily to support both `ipfs:<ipld>` and `/ipfs/<ipld>` so I'm not worried, but we should think about transitioning over the mid-term.

Reply:
Good point, yup, we can do that